src/Settings.php

```diff
@@ -647,6 +651,16 @@ public function admin_page_display( string $plugin_page ): void {
 		);
 	}
 
+	/**
+	 * Show notice to inform users that the Donate button settings are now in Appearance > Menus.
+	 */
+	public function show_donate_menu_notice(): void {
```

Comment:
I came up with the copy for this on my own when working on the ticket, but probably someone (maybe @comzeradd?) should review it 😬

Reply:
Maybe we also need documentation for this 🤔 It is a bit complicated since you need to create a new menu etc

Reply:
Actually just saw that there is [documentation](https://planet4.greenpeace.org/manage/admin-setup/#donate-button) for the Donate button already, that will probably need to be updated?